requirements/models.txt

```diff
@@ -0,0 +1 @@
+ultralytics==8.3.162
```

Comment:
ultralytics appears to have almost daily releases. For the sake of both of our sanity, let's do what we did for setuptools in `.github/dependabot.yml`